client/src/components/MenuItem.tsx

```diff
@@ -14,43 +15,27 @@ export const MenuItem = ({
   itemName: string;
   active: boolean;
 }) => {
-  /*
-   * Store our anchorTarget in state
-   * We do not set it here, preferring to wait for after the component
-   * is mounted to avoid any errors
-   */
-  const [anchorTarget, setAnchorTarget] = useState<HTMLElement | null>(null);
-
-  /*
-   * When the component mounts and/or updates, set our AnchorTarget based
-   * on the itemName
-   */
-  useEffect(() => {
-    setAnchorTarget(document.getElementById(itemName));
-  }, [itemName]);
-
-  /*
-   * Where all the magic happens -- scrollIntoView on click
-   */
-  const handleClick = (event: any) => {
-    event.preventDefault();
-    anchorTarget?.scrollIntoView({ behavior: "smooth", block: "start" });
-  };
-
-  let name = "";
+  let name = '';
   switch (itemName) {
-    case "number-of-txs":
-      name = "Number of transactions";
+    case 'overview':
```

Comment:
As now we track the active state of the menu via pages, we can see on which page we are from the next.js params, probably this file can be greatly simplified/removed.